src/scss/root/_index.scss

```
@@ -214,6 +214,13 @@ $component-map: get-all-component-definitions($tokens, $breakpoints);
 		}
 	}
 }
+:root {
+	--text-direction: 1;
+}
+
+html[dir="rtl"] {
```

```suggestion
*[dir="rtl"] {
```
This is mainly for storybook where we've been using the <div dir="rtl">